modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -43,7 +44,7 @@ def submit # rubocop:disable Metrics/MethodLength
 
           track_pact_counter auto_claim
           pdf_data = get_pdf_data
-          pdf_mapper_service(form_attributes, pdf_data, target_veteran).map_claim
+          pdf_mapper_service(form_attributes, pdf_data, veteran).map_claim
```

Comment:
Why veteran, rather than target_veteran?

Reply:
We had that call to target_veteran in two places, so I figured call it once and store in local var for reuse.

Reply:
that makes sense, and I am wondering if that adds confusion rather than simplification

Reply:
yeah.. i get what you're going for here, but the instance variable and the `||=` saves us from executing all that logic again. Sure, it will look to the `target_veteran` method but after the first run it will return the `@target_veteran` instead of running the `||=` block.

In v1 it actually executes the entire block over and over again.. v2's current structure prevents that and just stores it to that instance variable.

Reply:
Jennica & I were just discussing that...it's not saving anything because it's already stored in the class variable in application_controller, as you noted. I reverted the change & pushed up.

Reply:
and for clarity, we can't pass `target_veteran` to the future sidekiq jobs because it will log the PII, so the latter call to that method will be moved internal of a job anywho.....

in the long run, this won't be an issue because of that.